crates/ty_python_semantic/src/types.rs

```diff
@@ -7849,6 +7862,7 @@ impl<'db> KnownInstanceType<'db> {
                     }
                     KnownInstanceType::UnionType(_) => f.write_str("types.UnionType"),
                     KnownInstanceType::Literal(_) => f.write_str("typing.Literal"),
+                    KnownInstanceType::Annotated(_) => f.write_str("typing.Annotated"),
```

Comment:
I'm not wild about the displays for these two, because `int` as a type display means "instance of `int`", and `list[int]` means "inhabitant of the type `list[int]`"... but `KnownInstance` types here that would be displayed as `typing.Literal` or `typing.Annotated` aren't actually instances/inhabitants of `typing.Literal` or `typing.Annotated`. Neither of those symbols are classes at runtime, so it would be impossible to create an "instance of `typing.Literal`".

What about something like `<subscripted typing.Literal>` or `<typing.Literal special form>`?